### PR TITLE
Add renew now functionality to dashboard

### PR DESCRIPTION
**Pull request by penoptech**
This PR introduces a new feature to the dashboard, allowing users to renew their applications on-demand. Previously, users had to wait for the renewal period to open or contact the City to initiate a renewal. This feature aims to improve user autonomy and satisfaction by enabling them to renew their applications whenever they choose, directly from their dashboard.

### CODE REVIEW

File: getyour/manage.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'getyour.settings.dev')
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'getyour.settings.local')
```
Comment (TiCampbell) [inline getyour/manage.py:9]: Our test Azure app instance relies on the default of `getyour.settings.dev`.

**Review by TiCampbell — CHANGES_REQUESTED** [review]

File: getyour/app/views/application.py [hunk 2]
```diff
@@ -1150,7 +1150,7 @@ def household_members(request, **kwargs):
                     uploaded_file = SimpleUploadedFile(
                         file_name, buffer.read(), content_type)
                     file_paths.append(file_path)
-                    default_storage.save(file_path, uploaded_file)
+                    # default_storage.save(file_path, uploaded_file)
```
[the remark this answers is not in the document]
Comment (penoptech) [reply getyour/app/views/application.py]: Ack! Yes, I forgot to undo some of my local dev changes. Thanks for catching this!

File: getyour/app/views/application.py [hunk 3]
```diff
@@ -1429,14 +1429,14 @@ def files(request, **kwargs):
...
                     # afterward
-                    instance.document_path.save(
-                        pendulum.now(
-                            'utc'
-                        ).format(
-                            f"YYYY-MM-DD[T]HHmmss[Z_{fileAmount}_{f}]"
-                        ),
-                        f,
-                    )
+                    # instance.document_path.save(
```
[the remark this answers is not in the document]
Comment (penoptech) [reply getyour/app/views/application.py]: Yeah, same thing here. Just for testing. This change has been rolled back now

**Comment by penoptech** [comment]
@TiCampbell - Apologies! I have a local setup for development, and I forgot to exclude them from this commit and PR. Changes to `local.py`, `manage.py`, and `application.py` have all been rolled back to their original state. Thanks for catching that!

File: getyour/manage.py (hunk 1 above)
Comment (penoptech) [reply getyour/manage.py]: Same issue here with my local dev setup

File: getyour/getyour/settings/local.py
[the remark this answers is not in the document]
Comment (penoptech) [reply getyour/getyour/settings/local.py:1]: I develop locally with a containerized Postgres database. I reverted this change as well.

**Comment by TiCampbell** [comment]
This has been merged into `dev`, migrations applied, `renewal_interval_year` set properly for the applicable `IQProgramRD` objects, and deployed to the Get FoCo test site.

**Pull request merged**